docs/development/codeguide.rst

```diff
@@ -490,50 +482,6 @@ the hierarchy.
 .. note:: For more information on the benefits of `super`, see
           https://rhettinger.wordpress.com/2011/05/26/super-considered-super/
 
-.. _import-star-example:
-
-Acceptable use of ``from module import *``
```

Comment:
Should we at least keep this section and mention that it is no longer allowed except in `__init__.py`? Otherwise we might have new contributors who write a new subpackage or something and then wonder why ruff is blocking them. Better they know from the start while reading the guide?

Also if for whatever reason, people link to `import-star-example` ref from outside, it will now be broken.

Reply:
I think it's a reasonable Q. But I think new devs will see wildcards frequently in other `__init__.py` and / or a maintainer will suggest the shortening. IMO we can remove the verbiage.